firmware/user/modes/mode_tunernome.c

```diff
@@ -169,10 +172,41 @@ const uint16_t freqBinIdxs[NUM_STRINGS] =
     24  // A string is exactly at note #24
 };
 
+/**
+ * Indicies into fuzzed_bins[], a realtime DFT of sorts
+ * fuzzed_bins[0] = A ... 1/2 steps are every 2.
+ */
+const uint16_t freqBinIdxsUkelele[NUM_STRINGS] =
+{
+    68, // G
+    72, // A
+    62, // E = C + 4 half steps = 62
+    54  // C string is exactly at note #54
+};
+
+const char* semitoneNoteNames[12] =
+{
+    "C",
+    "C#",
+    "D",
+    "D#",
+    "E",
+    "F",
+    "F#",
+    "G",
+    "G#",
+    "A",
+    "A#",
+    "B"
+};
+
 static const char tn_title[]  = "Tunernome";
+static const char theWordGuitar[] = "Guitar";
 static const char leftStr[] = "< Exit";
 static const char rightStrTuner[] = "Tuner >";
 static const char rightStrMetronome[] = "Metronome >";
+static const char upArrowStr[] = "/\\";
```

Comment:
1. Make a png, put it in the `assets` folder
1. Delete all `assets.bin` files so it gets regenerated on build
1. Call `allocPngAsset()` to load the image (mode enter)
1. Call `drawPng()` to draw the image. You can even have one arrow and just rotate it!
1. Call `freePngAsset()` to free it when done (mode exit)

Reply:
You could also use that arrow for left/right with more rotations too